src/locales/de/ErrorPage.json

```diff
@@ -1,4 +1,4 @@
 {
     "Subtitle": "Seite nicht gefunden",
-    "MainText": "Sie haben unsere Fehlerseite gefunden. Dies ist nicht Ihre Schuld, sondern ein Fehler auf unserer Serverseite. Bitte entschuldigen Sie die entstandenen Unannehmlichkeiten und kontaktieren Sie gern unser Team unter dem Reiter 'Probleme melden'."
+    "MainText": "Diese Seite konnte nicht gefunden werden. Vielleicht besteht ein Fehler in der eingegebenen URL oder auf unserer Serverseite. Bitte entschuldigen Sie die entstandenen Unannehmlichkeiten und kontaktieren Sie gern unser Team unter dem Reiter 'Probleme melden'."

```

Comment:
I would remove the "Bitte entschuldigen ... Unannehmlichleiten" part too

Reply:
It is only a text proposal. Katja (and Dominic) will change it again anyway.